tests/testutils/endpoint.go

```diff
@@ -78,8 +79,9 @@ func GetAvailablePort(t testing.TB) uint16 {
 
 	portInt, err := strconv.Atoi(port)
 	require.NoError(t, err)
-
-	return uint16(portInt)
+	require.Greater(t, portInt, -1)
+	require.Less(t, portInt, math.MaxUint16+1)
+	return uint16(portInt) //nolint:gosec
```

Comment:
This is in test code. Made safe to ignore the lint on this line by checking if the `int` is on the range of a "good" cast to `uint64`.